packages/rocketchat-api/server/routes.coffee

```diff
@@ -110,3 +110,26 @@ RocketChat.API.v1.addRoute 'groups.list', authRequired: true,
 	get: ->
 		roomIds = _.pluck RocketChat.models.Subscriptions.findByTypeAndUserId('p', @userId).fetch(), 'rid'
 		return { groups: RocketChat.models.Rooms.findByIds(roomIds).fetch() }
+
+# List all users
+RocketChat.API.v1.addRoute 'users.list', authRequired: true,
+	get: ->
+		return { users: RocketChat.models.Users.find().fetch() }
```

Comment:
Please test with a DB with 100k (generated) user entries.   This can become an attack vector due to the low auth requirements of a users-list.   Thanks.